h2o-core/src/main/java/water/api/schemas3/ParseV3.java

```diff
@@ -69,4 +70,7 @@
   // Zero if blocking==false; row-count if blocking==true
   @API(help="Rows", direction=API.Direction.OUTPUT)
   public long rows;
+
+  @API(help="One-character string used to escape other characters.", direction=API.Direction.INOUT)
+  public byte escape_char = ParseSetup.DEFAULT_ESCAPE_CHAR;
```

Comment:
Should we actually make it a String in the API and then validate its properties?

Reply:
can be, I did it like byte since separator is also byte

Reply:
then it is probably better to be consistent, was just thinking out loud